_rules/audio-transcript-2eb176.md

```diff
@@ -5,6 +5,12 @@ rule_type: atomic
 description: |
   This rule checks that `audio` elements have a transcript that includes all auditory information.
 accessibility_requirements:
+  wcag20:1.2.1: # Audio-only and Video-only (Prerecorded) (A)
+    forConformance: true
+    secondary: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
```

Comment:
Iirc, we do not have mappings for secondary requirements 🤔

Reply:
As above